LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This should be editing the existing `LICENSE.md` rather than adding a new `LICENSE` file?

Reply:
Yes thats correct.